wiki/Beatmapping/Health_Drain/en.md

```diff
@@ -1,7 +1,99 @@
 
-<!--TODO: Insert links to health bar, difficulty setting, mapper, ranked, unranked, modes -->
\ No newline at end of file
+The HP difficulty settting controls how heavily a player is penalized for missing notes and how much health is gained back by accurately hitting [hit objects](/wiki/Hit_Objects). HP values range from HP0 to HP10, where HP10 is the highest and most punishing setting and HP0 the most forgiving. The HP difficulty setting can be changed alongside other difficulty settings in the [Beatmap Editor](/wiki/Beatmap_Editor). Doing this will change a Ranked beatmap's status to unranked and will erase all local scores on it.<!--TODO: might be worth writing about this more in depth-->
+
+## Health drain
+
+A common system across game modes is known as health drain, which refers to the natural drain of the health bar that occurs when playing a beatmap. This value is also known as *HP drain*, and is the one mostly referred to when talking about health. Health drain is dependent on the HP difficulty setting, which is set by the mapper of a beatmap. HP values range from 0 to 10, where 10 is the highest and most punishing setting and 0 the most forgiving. On HP10 the health drains rapidly and a single miss may be enough to fail a player, whereas on HP0 multiple combos can be missed before failing a player.
+
+## osu!standard
+
+In osu!standard health will decrease at a steady rate (passive health drain), but can be replenished by tapping notes at the right time or spinning spinners. Score bursts will appear after tapping hit circles, changing score and health values depending on the accuracy of the hit. A perfectly timed hit (300) will recover health more than a badly timed hit (50). The player also receives larger boosts if he achieves a [katu](#katu) or a [geki](#geki) at the end of a combo. A miss will drain health instead. If spinners are cleared early, continuing to spin will gain additional health back. For an overview of health recovery and health loss see below:
+
+Health recovery:
+
+- A successful tap on the hit circle (better judgement leads to more health gain; the last hit circle in a colour combo awards more health).
+- A successful tap on a slider start.
+- The slider ball collected a slider tick successfully.
+- The slider ball touched the slider end or a reverse arrow successfully (better judgement at the end tick leads to more health gain).
+- Completing a spinner and continuous spinning in the spinner duration (better judgement at the end of the spinner leads to more health gain).
+
+Health loss:
+
+- Passive health drain.
+- Missing a hit circle.
+- Tapping a slider head early.
+- Missing a slider tick, or slider reverse arrow.
+- Failed to achieve at least a 50 on a spinner.
+
+Health reserved:
```

Comment:
I would probably just make a new sentence here (or better yet at the start) mentioning that health drain won't take effect before the start of the beatmap or during breaks in any scenario, since I think this applies to all modes?

Reply:
This serves more as an overview of ALL things that apply to a game mode, so that everything that affects a game mode is easily accessible. This leaves no room for confusion.

but i agree it's definitely worth mentioning in the health drain section